packages/espressocash_app/lib/features/tokens/data/token_repository.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:drift/drift.dart';
+import 'package:injectable/injectable.dart';
+
+import '../../../data/db/db.dart';
+
+@singleton
+class TokenListRepository {
+  const TokenListRepository(this._db);
+
+  final MyDatabase _db;
+
+  Future<List<TokenRow>> getAllTokens() => _db.select(_db.tokenRows).get();
+
+  Stream<List<TokenRow>> watchAllTokens() => _db.select(_db.tokenRows).watch();
```

Comment:
There shouldn't be methods like that, we don't want to load thousands of tokens into the memory.